renku/cli/workflow.py

```diff
@@ -151,25 +162,82 @@ def rename(old, new, force):
 
 @workflow.command()
 @click.argument("name", metavar="<name>")
-def remove(name):
-    """Remove the remote named <name>."""
-    remove_workflow_command().build().execute(name=name)
+@click.option("--force", is_flag=True, help="Override the existence check.")
+def remove(name, force):
+    """Remove a workflow named <name>."""
+    remove_workflow_command().build().execute(name=name, force=force)
+
+
+def _kv_to_dict(ctx, param, values):
+    return dict([value.split("=") for value in values])
 
 
 @workflow.command()
-@click.option("--revision", default="HEAD")
+@click.argument("workflow_name", metavar="<name or uuid>")
+@click.option("--name", metavar="<new name>", help="New name of the workflow")
+@click.option("--description", metavar="<new desc>", help="New description of the workflow")
+@click.option(
+    "--set",
+    "set_params",
+    multiple=True,
+    metavar="<parameter>=<value>",
+    callback=_kv_to_dict,
+    help="Set default <value> for a <parameter>/add new parameter",
+)
+@click.option(
+    "--map", "map_params", metavar="<parameter>=<parameter or expression>", help="New description of the workflow"
+)
+@click.option(
+    "--rename-param",
+    "rename_params",
+    multiple=True,
+    metavar='<parameter>="name"',
+    callback=_kv_to_dict,
+    help="New description of the workflow",
+)
+@click.option(
+    "--describe-param",
+    "describe_params",
+    multiple=True,
+    metavar='<parameter>="description"',
+    callback=_kv_to_dict,
+    help="New description of the workflow",
+)
+def edit(workflow_name, name, description, set_params, map_params, rename_params, describe_params):
+    """Edit workflow details."""
+    edit_workflow_command().build().execute(
+        name=workflow_name,
+        new_name=name,
+        description=description,
+        set_params=set_params,
+        map_params=set_params,
+        rename_params=rename_params,
+        describe_params=describe_params,
+    )
+
+
+@workflow.command()
+@click.argument("workflow_name", metavar="<name or uuid>")
+@click.option(
```

Comment:
I know it's not in the spec, an oversight on my part, but we probably want to allow a user to specify a values file (like `--values values.yml`) as in the `renku workflow execute` spec, so they can export with value overrides. You should be able to just use the method I wrote in the other PR `apply_run_values` and pass it as a dict after loading from yaml.